app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,12 @@
   </head>
```

Comment:
- [ ] As the previous reviewer mentioned kindly make sure the app redirects to the login page, as for now it shows a blank page